.github/workflows/stale.yml

```diff
@@ -13,7 +13,11 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         days-before-stale: 60
         days-before-close: 7
-        stale-issue-message: 'This issue has been marked as stale because it has been open for a while and has had no recent activity. If this issue is still important to you please drop a comment below and we will add this to our backlog to complete. Otherwise, it will be closed in 7 days.'
-        stale-issue-label: 'stale'
-        stale-pr-message: 'This PR has been marked as stale because it has been open for a while and has had no recent activity. If this PR is still important to you please drop a comment below and we will add this to our backlog to complete. Otherwise, it will be closed in 7 days.'
-        stale-pr-label: 'stale'
+        stale-issue-message: 'Hi, this issue has been open for a while and has had no recent activity so it has been labeled for attention. With significant change within our team over the past 6 months, we are no longer aware if your issue is still relevant and/or up-to-date.
```

Comment:
I don't think we need to mention the team changes. This bot is going to be something that is around for a while so lets future proof the message.

Reply:
I thought we were only going to use this bot for a short time period until we got the workload under control. Afterwards, i dont think 7 days is enough time for a brand new PR to become outdated (nor 30 or 60 days for that matter)